test/stopAfterParseTests.sh

```diff
@@ -70,5 +70,5 @@ while read -r file; do
 		echo "$file"
 		exit 1
 	fi
-done < <(find "${REPO_ROOT}/test" -iname "*.sol" -and -not -name "documentation.sol")
+done < <(find "${REPO_ROOT}/test" -iname "*.sol" -and -not -name "documentation.sol" -and -not -name "boost_filesystem_bug.sol")
```

Comment:
I'd suggest to split it into mutliple lines so that we can explain why each of these files is ignored.
```suggestion
done < <(
    find "${REPO_ROOT}/test" -iname "*.sol"
        -and -not -name "documentation.sol"
        # This test should not be passed through splitSources.py because it's specifically crafted to
        # use an invalid file name in its header (`==== Source://// ====`).
        -and -not -name "boost_filesystem_bug.sol"
)
```

Reply:
The shellchecker complains that a multi line find command is somehow problematic :(

Reply:
Oh. ok then. I test locally that it worked when I suggested it but I didn't run shellcheck on it so maybe there is something wrong about it after all.